Base/usr/share/man/man1/realsize.md

```diff
@@ -0,0 +1,40 @@
+## Name
+
+realsize - Show the 'real' size of a number with a suffix (possibly).
+
+## Synopsis
+
+```**sh
+$ realsize [integer-with-suffix]
+```
+
+## Description
+
+`realsize` prints the 'real' size of a number with a suffix (possibly).
+It can parse just one-char suffix (k for kilo, for example), or base 2 or 10
```

Comment:
Typo: just -> just a (personally I would write: "It can parse just a single character suffix", but :shrug:)

Also, it won't parse a lower case 'k' suffix at the moment.